language/move-lang/tests/move_check/parser/byte_string.move

```diff
@@ -0,0 +1,12 @@
+module M {
+    public fun byte_string(): vector<u8> {
+        b"Libra"
+    }
+    public fun byte_string_with_quotes(): vector<u8> {
+         b"\"Libra\""
+    }
+    public fun multi_line_byte_string(): vector<u8> {
+             b"Hello
+             Libra"
```

Comment:
How is this handled exactly? Does it include all of the whitespace in the string?

Reply:
literal behavior will be like in rust.